test/vcf/optional_policy_test.cpp

```diff
@@ -135,6 +135,51 @@ namespace ebi
                                 vcf::InfoBodyError*);
             }
         }
+
+        SECTION("SVLEN test")
+        {
+                source->meta_entries.emplace(vcf::INFO,
+                    vcf::MetaEntry{
+                        1,
+                        vcf::INFO,
+                        {
+                            { vcf::ID, vcf::SVLEN },
+                            { vcf::NUMBER, "2" },
+                            { vcf::TYPE, vcf::INTEGER },
+                            { vcf::DESCRIPTION, "Difference in length between REF and ALT alleles" }
+                        },
+                        source
+                });
+
+                CHECK_NOTHROW( (optional_policy.optional_check_body_entry(parsing_state, vcf::Record{
+                                    1,
+                                    "chr1",
+                                    123456,
+                                    { "id123" },
+                                    "A",
+                                    { "AC","AT" },
```

Comment:
Change this to `{ "AC", "ACT" }`.
SVLEN is the difference between the length of ALT and REF columns (as you can see in the Description you added in line 149). To ensure that this does not throw, you should change the values.
You could change other things as well, like the REF column or SVLEN.